consume_messages.rb

```diff
@@ -0,0 +1,38 @@
+require 'aws-sdk'
+
+# Bring environment variables in ./config/.env into scope.
+# This is probably only used in development.
+require 'dotenv'
+Dotenv.load(File.join('.', 'config', '.env'))
+
+# Create a hash that contains the configuarable variables
+require 'erb'
+require 'yaml'
+path_to_settings = File.join(__dir__, "config", "settings.yml")
+settings = YAML.load(ERB.new(File.read(path_to_settings)).result)
+
+# Configure SQS Client
+credentials = Aws::Credentials.new(settings['aws_key'], settings['aws_secret'])
+sqs_client  = Aws::SQS::Client.new(region: 'us-east-1', credentials: credentials)
+
```

Comment:
regarding `attribute_names`....

[Attributes](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-message-attributes.html) are like headers that come with the message.

I experimented with adding a message to the queue via the AWS web interface and I'm 90% sure that the
headers [in this gist](https://gist.github.com/nodanaonlyzuul/479cf45e95668851696a63540ce3f726) are part of a message, automatically, regardless of how the message was created (some Ruby/Python/JS library, or the web console).

Reply:
👍